packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -102,12 +102,36 @@ def config_all_loggers(*, log_format_local_dev_enabled: bool) -> None:
         fmt = LOCAL_FORMATTING
 
     for logger in loggers:
-        set_logging_handler(
+        _set_logging_handler(
             logger, fmt=fmt, log_format_local_dev_enabled=log_format_local_dev_enabled
         )
 
```

Comment:
Idea here is that `LOGGER_FILTER_MAPPING` will be taken from the Application settings and passed to the `config_all_loggers` function similar to `log_format_local_dev_enabled`. What do you think about this approach?

Reply:
yes I like it.
If I get it right you will create some layered approach right?

the uvicorn ones for uvicorn-based services, the gunicorn for gunicorn-based ones?
We might also incorporate the NOISY_LOGGERS, which we start having all over the place, and there we could also have some list that can be layered as we have a few we know are especially noisy (aiopika to name a few)?

now I am a tiny little bit concerned about performance of that thing, since we log millions of logs this can have an impact. but we can test, maybe the jaeger can help to find out.

Reply:
Okay, I have changed it to an environment variable in my last commit. 👍 Correct, we can extend this and also add NOISY_LOGGERS as the next step. Regarding performance, we can test it once it's in the master branch. By default, it's an empty dictionary. Performance should be "worsen" only for the loggers that are provided in the ENV var as only for those filtering is happening.